adopters.md

```diff
@@ -7,31 +7,71 @@ category: adopters
+|[LY.com](https://www.crunchbase.com/organization/ly-com)|Travel|[Chinese](https://www.pingcap.com/cases-cn/user-case-tongcheng/)||
+|[Qunar.com](https://www.crunchbase.com/organization/qunar-com)|Travel|[Chinese](https://www.pingcap.com/cases-cn/user-case-qunar/)|
+|[Hulu](https://www.hulu.com/welcome?orig_referrer=https%3A%2F%2Fwww.google.com%2F)|Entertainment|||
+|[VIPKID](https://en.wikipedia.org/wiki/VIPKID)|EdTech|||
+|[Lenovo](https://en.wikipedia.org/wiki/Lenovo)|Enterprise Technology|||
+|[Bank of Beijing ](https://en.wikipedia.org/wiki/Bank_of_China)|Banking|||
+|[Industrial and Commercial Bank of China](https://en.wikipedia.org/wiki/Industrial_and_Commercial_Bank_of_China)|Banking|||
+|[iQiyi](https://en.wikipedia.org/wiki/IQiyi)|Media and Entertainment|||
+|[Yimian Data](https://www.crunchbase.com/organization/yimian-data)|Big Data|[Chinese](https://www.pingcap.com/cases-cn/user-case-yimian)|||
+|[Phoenix New Media](https://www.crunchbase.com/organization/phoenix-new-media)|Media|[Chinese](https://www.pingcap.com/cases-cn/user-case-ifeng/)||
+|[Mobikok](http://www.mobikok.com/en/)|AdTech|[Chinese](https://pingcap.com/cases-cn/user-case-mobikok/)||
+|[LinkDoc Technology](https://www.crunchbase.com/organization/linkdoc-technology)|HealthTech|[Chinese](https://www.pingcap.com/cases-cn/user-case-linkdoc/)||
+|[G7 Networks](https://www.english.g7.com.cn/)| Logistics|[Chinese](https://www.pingcap.com/cases-cn/user-case-g7/)||
+|[360 Finance](https://www.crunchbase.com/organization/360-finance)|FinTech|[Chinese](https://www.pingcap.com/cases-cn/user-case-360/)||
+|[GAEA](http://www.gaea.com/en/)|Gaming|[Chinese](https://www.pingcap.com/cases-cn/user-case-gaea-ad/)|[English](https://www.pingcap.com/blog/2017-05-22-Comparison-between-MySQL-and-TiDB-with-tens-of-millions-of-data-per-day/)|
+|[YOOZOO Games](https://www.crunchbase.com/organization/yoozoo-games)|Gaming|[Chinese](https://pingcap.com/cases-cn/user-case-youzu/)||
+|[Seasun Games](https://www.crunchbase.com/organization/seasun)|Gaming|[Chinese](https://pingcap.com/cases-cn/user-case-xishanju/)||
+|[NetEase Games](https://game.163.com/en/)|Gaming|||
+|[FUNYOURS JAPAN](http://company.funyours.co.jp/)|Gaming|[Chinese](https://pingcap.com/cases-cn/user-case-funyours-japan/)||
+|[Zhaopin.com](https://www.crunchbase.com/organization/zhaopin)|Recruiting|||
+|[Panda.tv](https://www.crunchbase.com/organization/panda-tv)|Live Streaming|||
+|[Hoodinn](https://www.crunchbase.com/organization/hoodinn)|Gaming|||
+|[Ping++](https://www.crunchbase.com/organization/ping-5)|Mobile Payment|[Chinese](https://pingcap.com/cases-cn/user-case-ping++/)||
+|[Hainan eKing Technology](https://www.crunchbase.com/organization/hainan-eking-technology)|Enterprise Technology|[Chinese](https://pingcap.com/cases-cn/user-case-ekingtech/)||
+|LianLian Tech|Mobile Payment|||
```

Comment:
For LianLian Tech, Tree Finance, and 2Dfire.com, the website link is omitted. Is that on purpose?

Reply:
Good catches! For the website link, it was not included because there was no Engilsh name. But I think we can add the Chinese name instead.